tests/TestCase.php

```diff
@@ -34,7 +39,14 @@ public function createApplication()
         require $framework . '/Illuminate/Foundation/start.php';
         require __DIR__ . '/../src/Gzero/Core/helpers.php';
 
-        $this->app['artisan']->call('migrate', ['--path' => '../src/migrations']); // Relative to tests/app/
+        if (!self::$initialSetup) {
+            try {
+                $this->app['artisan']->call('migrate:reset');
```

Comment:
Added this handy block code to cover case when we updated migration that was already installed in test DB